bundles/io/org.eclipse.smarthome.io.net.test/pom.xml

```diff
@@ -15,23 +15,96 @@
 
   <name>Eclipse SmartHome Network I/O Tests</name>
 
-  <properties>
-    <bundle.symbolicName>org.eclipse.smarthome.io.net.test</bundle.symbolicName>
-    <bundle.namespace>org.eclipse.smarthome.io.net.test</bundle.namespace>
-  </properties>
-
   <build>
     <plugins>
       <plugin>
         <groupId>${tycho-groupid}</groupId>
         <artifactId>target-platform-configuration</artifactId>
         <configuration>
-          <environments combine.self="override"></environments>
+          <dependency-resolution>
+            <extraRequirements>
+              <requirement>
+                <type>eclipse-plugin</type>
+                <id>org.eclipse.equinox.ds</id>
+                <versionRange>0.0.0</versionRange>
+              </requirement>
+              <requirement>
+                <type>eclipse-plugin</type>
+                <id>org.eclipse.equinox.event</id>
+                <versionRange>0.0.0</versionRange>
+              </requirement>
+              <requirement>
+                <type>eclipse-plugin</type>
+                <id>org.eclipse.smarthome.core.thing</id>
+                <versionRange>0.0.0</versionRange>
+              </requirement>
+              <requirement>
+                <type>eclipse-plugin</type>
+                <id>org.eclipse.smarthome.core.voice</id>
```

Comment:
Are you sure that you need the voice, audio, etc. bundles as a dependency here?

Reply:
No, I have no idea why all other test bundles contain these, what I noticed was that it started all kinds unexpected processes like a plug wise discovery and I did not have this so I though I had to add it 😐